rules/no-todo-implementation.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const visitIf = require('enhance-visitors').visitIf;
+const createAvaRule = require('../create-ava-rule');
+
+const functionExpressions = [
+	'FunctionExpression',
+	'ArrowFunctionExpression'
+];
+
+function isFunction(node) {
+	return node &&
+		functionExpressions.indexOf(node.type) !== -1;
+}
```

Comment:
Can you move this (line 5-13) to https://github.com/avajs/eslint-plugin-ava/blob/master/util.js ?